Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6-alpine
+
+COPY requirements.txt /app/
+
+RUN true \
+	&& pip install gunicorn \
+	&& pip install -r /app/requirements.txt \
+	&& true
+
+WORKDIR /app/src
```

Comment:
See the comment above. The `iwant_bot` package is located in the `src` subdirectory.